symbology/apply-unique-values-with-alternate-symbols/src/main/java/com/esri/samples/apply_unique_values_with_alternate_symbols/ApplyUniqueValuesWithAlternateSymbolsSample.java

```diff
@@ -0,0 +1,231 @@
+      SymbolReferenceProperties blueSquareRef = new SymbolReferenceProperties(10000.0, 5000.0);
+      SymbolReferenceProperties yellowDiamondRef = new SymbolReferenceProperties(20000.0, 10000.0);
+      SymbolReferenceProperties redTriangleRef = new SymbolReferenceProperties(5000.0, 0.0);
+
+      mBlueSquareSymbol.setReferenceProperties(blueSquareRef);
+      mYellowDiamondSymbol.setReferenceProperties(yellowDiamondRef);
+      mRedTriangleSymbol.setReferenceProperties(redTriangleRef);
+
+      // Creating unique value for the red triangle and list of alternate symbols
+      UniqueValueRenderer.UniqueValue uniqueValue =
+        new UniqueValueRenderer.UniqueValue("unique values based on request type", "unique value",
+          mRedTriangleSymbol, List.of("Damaged Property"), List.of(mBlueSquareSymbol, mYellowDiamondSymbol));
+
+      // Creating unique value renderer
+      UniqueValueRenderer uniqueValRenderer = new UniqueValueRenderer();
+      uniqueValRenderer.getUniqueValues().add(uniqueValue);
+      uniqueValRenderer.getFieldNames().add(("req_type"));
+
+      // Applying default symbol for the unique value renderer.
+      uniqueValRenderer.setDefaultSymbol(mPurpleDiamondSymbol);
+
+      // creating a service feature table using the url
+      final ServiceFeatureTable featureTable = new ServiceFeatureTable(FEATURE_SERVICE_URL);
+
+      // creating a feature layer from the service feature table
+      featureLayer = new FeatureLayer(featureTable);
+
+      // setting the unique value renderer on the feature layer
+      featureLayer.setRenderer(uniqueValRenderer);
+
+      // waiting for the feature layer to load
+      featureLayer.loadAsync();
+
+      featureLayer.addDoneLoadingListener(()->{
+        if (featureLayer.getLoadStatus() == LoadStatus.LOADED) {
+          // adding feature layer to ArcGISMap
+          map.getOperationalLayers().add(featureLayer);
+        } else {
+          new Alert(Alert.AlertType.ERROR, "Error loading Feature Table from service" + featureLayer.getLoadError().getMessage()).show();
+        }
+      });
+
+      // adding the map view and UI elements to the stack pane
+      stackPane.getChildren().addAll(mapView, vBox);
+      StackPane.setAlignment(vBox, Pos.TOP_LEFT);
+      StackPane.setMargin(vBox, new Insets(10, 0, 0, 10));
+
+    } catch (Exception e) {
+      // on any error, display the stack trace.
+      e.printStackTrace();
+    }
+  }
+
+  /**
+   * Creates a UI with three buttons and a label.
+   * @return a vBox populated with buttons and labels
+   */
+  private VBox controlsVBox() {
+
+    // creating label to show scale values and button to reset viewpoint
+    lblScale = new Label("Current scale: 1:25000.00");
+    lblScale.setTextFill(Color.WHITE);
+    btnReset = new Button("Reset Viewpoint");
+
+    // creating and setting a box to hold the scale label and reset button
+    VBox vBox = new VBox(5);
```

Comment:
when the member variable is added this can become

```suggestion
    vBox = new VBox(5);
```